libretro/libretro.cpp

```diff
@@ -1310,7 +1330,18 @@ bool8 S9xDeinitUpdate(int width, int height)
 
 bool8 S9xContinueUpdate(int width, int height)
 {
-   return S9xDeinitUpdate(width, height);
+   if(interlace_speed==1)
+     return S9xDeinitUpdate(width, height);
+
+   else if(interlace_speed==2)
+     return true;
```

Comment:
why would one not want to render frames?

Reply:
No, what i mean is in interlace_speed=2 will just return true, and not pass to S9xDeinitUpdate() which renders the frame to screen...

Reply:
Oh. Uhm. Idk honestly. If we S9xDeinitUpdate, then we will get the half-frame updates upstream doesn't want to show normally. Not sure what to do about this. :o

Reply:
RPMR seems to run fine even in interlaced mode. but this works for you then go ahead. for me its just unnecessary to have it as an option if only a few games if not only this game suffers from it...

imho

Reply:
That's a good point! I like that thinking. The Chrono Trigger libretro hack doesn't need core option and quietly does something helpful. We can do that for Lufia 2 credits which is very very specific case.

Going to do that. Thanks! Shutting down PR and making clean one. Already goofed again with snes9x.h :(